### PR TITLE
Chore/admin

### PR DESCRIPTION
**Pull request by fr0gz**
Added django-admin

### CODE REVIEW

File: cmpirque/users/admin.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from django.contrib import admin
+from .models import User
+
+admin.site.register(User)
```
Comment (chivke) [inline cmpirque/users/admin.py:4]: you missing the blank line at the end

**Review by chivke — COMMENTED** [review]
Are you check for admin in the settings aps?

File: cmpirque/users/admin.py (hunk 1 above)
Comment (fr0gz) [reply cmpirque/users/admin.py:4]: i will put the blank line at the end

yes, i have checked the local settings according to documentation and it's ok

Comment (chivke) [reply cmpirque/users/admin.py]: Ok! and the tests passes?

**Review by chivke — APPROVED** [review]

**Pull request merged**